pages/glossary/visual-context.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Visual context
+key: visual-context
+---
+
+The visual context of some piece of information (_e.g._ an element) is everything that is visually or logically located near it. Only [visible](#visible) information may be part of the visual context.
```

Comment:
"near" is subjective. What does it mean for something to be "logically near" something? What is a "piece of information"?

Reply:
This is only used in the expectation, so subjective is OK.
"piece of information" has been replaced.